idea/data-storage/src/database/entities/meta.entity.ts

```diff
@@ -7,23 +7,23 @@ import { Program } from './program.entity';
 @Entity()
 export class Meta implements IMeta {
   @PrimaryGeneratedColumn('rowid')
-    id: string;
+  public  id: string;
 
   @Column()
-    program: string;
+  public program: string;
 
   @Column()
-    owner: string;
+  public owner: string;
 
   @Column({ nullable: true })
-    meta: string;
+  public hex: string;
 
-  @Column({ nullable: true })
-    metaWasm: string;
+  @Column({ nullable: true, type: 'json' })
+  public data: string;
```

Comment:
I'm not sure that `data` is the best option.
Maybe `types` or smth like this?

Reply:
Alright, `types` sounds better.